psalm-baseline.xml

```diff
@@ -15,36 +12,72 @@
       <code>int|false</code>
       <code>int|false</code>
     </InvalidReturnType>
-    <PossiblyInvalidArgument occurrences="1">
-      <code>$value</code>
-    </PossiblyInvalidArgument>
-  </file>
-  <file src="src/Collection/CollectionTrait.php">
-    <ArgumentTypeCoercion occurrences="2">
-      <code>$iterator</code>
-      <code>$iterator</code>
-    </ArgumentTypeCoercion>
-    <MissingParamType occurrences="1">
-      <code>$callback</code>
-    </MissingParamType>
-  </file>
-  <file src="src/Controller/ControllerFactory.php">
-    <ArgumentTypeCoercion occurrences="3">
-      <code>$request</code>
-      <code>$request</code>
-      <code>$request</code>
-    </ArgumentTypeCoercion>
   </file>
-  <file src="src/Http/BaseApplication.php">
-    <ArgumentTypeCoercion occurrences="1">
-      <code>$request</code>
-    </ArgumentTypeCoercion>
-  </file>
-  <file src="src/Routing/Middleware/RoutingMiddleware.php">
-    <ArgumentTypeCoercion occurrences="2">
-      <code>$request</code>
-      <code>$request</code>
-    </ArgumentTypeCoercion>
+  <file src="src/Core/Exception/CakeException.php">
+    <MethodSignatureMustProvideReturnType occurrences="62">
```

Comment:
oof, talk about unpaintable configs. can only generate this.

Reply:
This looks like a bug in psalm to me. It isn't reading the narrowed down return type in `@method` annotation correctly.